src/orb.yml

```diff
@@ -0,0 +1,335 @@
+version: 2.1
+description: "Utility orb for Coda Allows for: - Push alerts to opsgenie - Push Slack notifications - Cancel older CI alerts waiting for approval \n"
+display:
+    source_url: https://github.com/kr-project/circleci-utils
+commands:
+    cancel-older-awaiting-approvals:
+        description: |
+            Cancel all older pipelines waiting on manual approval.
+            Depends on jq being present.
+        parameters:
+            CIRCLE_TOKEN:
+                default: CIRCLE_TOKEN
+                description: |
+                    Env var of a token granted read access to the CircleCI api
+                type: env_var_name
+            vcs:
+                default: gh
+                description: |
+                    Defaults to "gh", but allows other version control systems.
+                type: string
+        steps:
+            - run:
+                command: |-
+                    #!/bin/bash
+                    set -eo pipefail
+
+                    BASE_URL="https://circleci.com/api/v2"
+                    TOKEN="circle-token=${CIRCLECI_API_TOKEN}"
+                    vcs="gh"
+                    echo "${BASE_URL}/workflow/${CIRCLE_WORKFLOW_ID}"
```

Comment:
Are the echo statements needed?   At a minimum, we shouldn't be leaking the token in the logs.